recipe/meta.yaml

```
@@ -17,7 +17,7 @@ build:
     - openai=openai.cli:main
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   # Minimum required version is 3.7.1, and s390x does not have pandas-stubs
-  skip: True # [py<37 or s390x]
+  skip: True # [py<38 or s390x]
```

```suggestion
  skip: True # [py<37 or s390x]
```
Upstream still supports 3.7. We should reflect that too.

---

Got it! Made the change